modules/simulations/batch-managed/modulestack.yaml

```diff
@@ -37,12 +42,12 @@ Resources:
             Action:
               - ecr:*
             Resource:
-              - !Sub "arn:aws:ecr:${AWS::Region}:${AWS::AccountId}:repository/addf-${DeploymentName}-${ModuleName}*"
+              - !Sub "arn:aws:ecr:${AWS::Region}:${AWS::AccountId}:repository/${ProjectName}-${DeploymentName}-${ModuleName}*"
           - Action:
               - "iam:CreateServiceLinkedRole"
```

Comment:
Can you confirm if you need this?

Reply:
Yap, we need to write to this ECR repo: https://github.com/awslabs/autonomous-driving-data-framework/blob/4bd8b3b1be065c4623ea15854caf20d90efbcc1d/modules/simulations/batch-managed/deployspec.yaml#L35